packages/client/src/components/SafeSettings/index.js

```diff
@@ -24,22 +25,35 @@ const SafeSettings = () => {
     removeVault,
     removeCollection,
   } = web3;
-
+  useEffect(() => {
+    (() => {
```

Comment:
Looks like it would work without the extra brackets:
```javascript
useEffect(() => {
    if (targetElementRef.current) {
      targetElementRef.current.scrollIntoView({ behavior: "smooth" });
    }
 //eslint-disable-next-line
 }, []);
  ```

Reply:
You are right. I got used to wrap async calls to IIFE, it's not needed here 👍

Reply:
Please check if you pushed this change, it's not reflected on the PR yet